pkg/cmd/scan.go

```diff
@@ -153,6 +155,12 @@ func NewScanCmd() *cobra.Command {
 		false,
 		"Includes cloud provider service-linked roles (disabled by default)",
 	)
+	fl.String(
+		"config-dir",
+		"",
+		".driftctl path to use.\n"+
+			"Default is home directory.\n",
+	)
```

Comment:
You can set the default value here directly, using `homedir.Dir()` or `os.Getenv("HOME")`

Reply:
i'm on it, it can be cool to avoid the logic in provider_installer.go